fixes typo in variable name for other files/media

pgmccann opened the pull request:
I came across this while trying to work out while our MP3 files weren't showing on the item pages. It appears that the variables $otherFiles and $otherMedia should be one and the same.

pgmccann committed:
fixes typo in variable name for other files/media

pgmccann committed:
fixes bug stopping non-image media viewer display

zerocrates commented:
On your second commit: enabling `other_media` is described as setting non-image media to display as links. So it's supposed to work the way it was: when that theme option is disabled you get the files actually rendered, when it's enabled you get the list of links. Your change there will make them not be displayed at all when other_media is off, and then both rendered and listed when it's on.

The title of the option is probably a little confusing and we might look at changing it.

pgmccann committed:
reverts previous - variable purpose misunderstood

Correction made in response to feedback https://github.com/omeka/theme-centerrow/pull/23#issuecomment-1270470333

pgmccann committed:
add alt attribute to img

An empty alt attribute is better than none at all.

pgmccann committed:
improves link accessibility

A difference in colour is insufficient to make links accessible

pgmccann committed:
adds title to iframe for viewer

pgmccann committed:
Increases contrast of file download links

pgmccann committed:
adds visual cue on focus on links and buttons

pgmccann committed:
makes the featured item carousel accessible

pgmccann committed:
addresses repeated element IDs accessibility issue

Having pagination links in twice results in repeated HTML element IDs, which isn't allowed

pgmccann committed:
stops seemingly random alignment of element names

pgmccann committed:
improves contrast for accessibility

pgmccann commented:
After fixing the [issue mentioned above](https://github.com/omeka/theme-centerrow/pull/23#issuecomment-1270470333) on 6 Oct in [8875aaf](https://github.com/omeka/theme-centerrow/pull/23/commits/8875aaf192e8719e781f294b73633e206dd38884), I've made a number of further changes, necessary for our site to meet legal accessibility requirements.

kimisgold commented:
Hi @pgmccann. Thanks for your accessibility edits. Many of them will be addressed as we incorporate the media viewer in the Omeka core. For the others, I've integrated them into the SASS so that they'll stick for users who compile their CSS. As such, I'm going to close this pull request. If after the next release you have additional notes on accessibility, I'm happy to discuss them further in subsequent issues or pull requests.
